### ISSUE TITLE
Generator do not create users service if authentication providers is local.

### ISSUE THREAD

**Issue by kulakowka** [feathersjs/generator-feathers#55]
When i attempted to create application with local authentication only, generator do not create `users service`.

![2016-02-25 22 52 42](https://cloud.githubusercontent.com/assets/557190/13332321/82abfc5e-dc12-11e5-86b6-7f6d41dc85c4.png)

![2016-02-25 22 53 11](https://cloud.githubusercontent.com/assets/557190/13332340/91b7c980-dc12-11e5-9aa6-aa1907cbeb5b.png)

The configuration file is no `token` options:
```javascript
{
  "host": "localhost",
  "port": 3030,
  "postgres": "postgres://postgres:@localhost:5432/test2",
  "public": "../public/"
}
```

This is normal behavior? How to create a user in this case?

When I generate an application with authentication via github (or any other oauth provider), the 'user' service is created.

**Comment by ekryski** [comment]
I know what that is. Thanks for reporting. Will fix later today.

**Comment by ekryski** [comment]
@kulakowka fixed in v0.6.6. Sorry about that.
